### PR TITLE
Add some backward compatibility for PHP 5.3 and 5.4 versions

### PR DESCRIPTION
**Pull request by Darkkin**

### CODE REVIEW

File: src/Client.php [hunk 1]
```diff
@@ -326,7 +326,25 @@ public function generatePrivateChannelToken($client, $channel, $exp = 0, $info =
         return implode('.', $segments);
     }
 
-    private function send($method, $params = [])
+/*
+ * Function added for backward compatibility with PHP version < 5.5
+ */
+    
+    private function json_last_error_msg() {
```
Comment (FZambia) [inline src/Client.php]: Do I understand right that on PHP >5.4 will use this function too instead of native one?

**Comment by Darkkin** [comment]
Yes, you're right. Do you think it's better to create a wrapper function with function_exists() check?

**Comment by FZambia** [comment]
@Darkkin yes, just to inherit all changes from this function in new PHP versions and fallback to this one only on old PHP versions.

**Comment by Darkkin** [comment]
Request updated. Please review added wrapper.

File: src/Client.php [hunk 2]
```diff
@@ -326,12 +326,33 @@ public function generatePrivateChannelToken($client, $channel, $exp = 0, $info =
...
     }
 
-    private function send($method, $params = [])
+/*
+ * Function added for backward compatibility with PHP version < 5.5
+ */
+    
+    public function _json_last_error_msg() {
+      if (function_exists('json_last_error_msg')) {
+        return json_last_error_msg(json_last_error());
```
Comment (FZambia) [inline src/Client.php]: Looking at https://www.php.net/manual/ru/function.json-last-error-msg.php json_last_error_msg definition is:
```php
json_last_error_msg ( void ) : string
```

Here you are calling it with argument `json_last_error()` - can you explain why?

Comment (Darkkin) [reply src/Client.php]: It's my mistake. Fixed.

**Pull request merged**